readme.md

```diff
@@ -182,6 +182,8 @@ Returns `true` if `value` is an odd integer.
 
 Returns `true` if `value` is falsy or an empty string, array, object, map, or set.
 
+##### .emoji(value)
```

Comment:
I'd prefer to see this _below_ `.emptyOrWhitespace` in the docs to:

1. Keep `.empty` and `.emptyOrWhitespace` together since they are similar in intent
2. Keep the order consistent with the source

Examples and / or a description would also be beneficial.